include/experimental/__p2642_bits/layout_padded.hpp

```diff
@@ -203,7 +204,7 @@ class layout_left_padded<PaddingValue>::mapping {
   }
 
 public:
-#if !MDSPAN_HAS_CXX_20
+#if !MDSPAN_HAS_CXX_20 || defined(__NVCC__)
```

Comment:
What's the problem here?

Reply:
NVCC does not like the requires stuff down there. Basically, it ends up saying that there is not default constructor on the device available in C++20 mode.

Reply:
I wasn't able to narrow it down in a simple reproducer.